uber/templates/preregistration/group_members.html

```diff
@@ -124,27 +124,45 @@
 
 {% if c.PRE_CON and group.min_badges_addable %}
     <div id="add" style="display:none">
-        <form method="get" action="add_group_members">
+      <form class="form-inline" method="get" action="add_group_members">
         <input type="hidden" name="id" value="{{ group.id }}" />
-        Enter a number of {{ group.is_dealer|yesno("Dealer Assistants,group members") }} to add{% if group.min_badges_addable > 1 %} (you cannot add fewer than {{ group.min_badges_addable }} badges to an existing group){% endif %}: <br/>
-        {% if group.is_dealer %}
-            <select name="count">{{ int_options(group.min_badges_addable, group.dealer_badges_remaining) }}</select>
-        {% else %}
-            <select name="count">{{ int_options(group.min_badges_addable, 10) }}</select>
-        {% endif %}
-        <input type="submit" value="{{ group.is_dealer|yesno("Add Dealer Assistants,Add Group Members") }}">
-        </form>
+        <p>
+          {%- set min_badges = group.min_badges_addable -%}
+          {%- set members = group.is_dealer|yesno("Dealer Assistants,group members") -%}
+          Enter the number of {{ members }} to add.
+          {% if group.min_badges_addable > 1 -%}
+            {%- set hours_remaining = group.hours_remaining_in_grace_period -%}
+            {%- if hours_remaining > 0 -%}
+              {%- set min_badges = 1 -%}
+              You have {{ humanize_timedelta(hours=hours_remaining, granularity='minutes') }}
```

Comment:
If I read the custom tag right, this would display something like "5 hours 45 minutes," right?